modules/ossm-enabling-sidecar-injection.adoc

```diff
@@ -0,0 +1,275 @@
+// Module included in the following assemblies:
+// install/ossm-sidecar-injection
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-enabling-sidecar-injection_{context}"]
+= Enabling sidecar injection
+
+The sidecar injection process in {istio} uses two labels to determine if a pod should be injected with a sidecar:
+
+* `istio-injection`
+* `istio.io/rev`
+
+Only one of these labels is required to enable injection. When both labels are applied, the `istio-injection` label takes precedence and treats the namespace as part of the default revision.
+
+By default, all pods are injected with a sidecar if injection is enabled at the namespace level. To disable injection for specific pods, set either the `istio-injection` label or the `sidecar.istio.io/inject` label to `disabled`. Alternatively, pods can be injected by enabling any of the following labels:
```

Comment:
Not true, `istio-injection` label can't be used to disable injection for a specific pod